src/main/java/com/exactpro/th2/infraoperator/operator/manager/impl/NamespaceEventHandler.java

```diff
@@ -112,7 +114,7 @@ public void eventReceived(Action action, Namespace resource) {
             try {
                 lock.lock();
 
-                logger.debug("Processing {} event for namespace: \"{}\"", action, namespaceName);
+                logger.info("Processing {} event for namespace: \"{}\"", action, namespaceName);
```

Comment:
question: How many events do we receive here? Won't it be too much if it is reported at the INFO level?

Reply:
This logging related to delete name space. This operation is performed rarely